### ISSUE TITLE
Table has no indexes

### ISSUE THREAD

**Issue by TeodorKolev** [esvit/ng-table#637]
For example if I want to add count my records when I go to second page and I have add $index it starts from 1 not continues to 11. Really a miss!

**Comment by hendricius** [comment]
Can you show a quick example? Thanks.

**Comment by TeodorKolev** [comment]
http://plnkr.co/edit/W39ctqy9ibGXwFbtrODt?p=preview
first  <td>{{$index+1}}</td> is index, so page 2 starts from 1 not from 11

**Comment by hendricius** [comment]
Thanks for the example. I just checked and there is no way yet to access the index when you paginate right now. You will have to implement this in your own business-logic in the data.

**Comment by TeodorKolev** [comment]
Does that means that I must implement it on Database? Is there an option to run for cycle or something in javascript and make that proper?

**Comment by TeodorKolev** [comment]
Also I want to get count in Group, and shows me again only items in page. I want to show me all items in group. Is that an option for that.

**Comment by hendricius** [comment]
Yep, I would do it in JS, not in the database. I would implement it in the `getData` function. There you have the info and the ordering and so on. Have to add it before you add the pagination.

Count in group - how would you like to do that? Do you have nested objects on each of the javascript objects? Or do you send data server-side grouped?

**Comment by TeodorKolev** [comment]
I want to count all items in group. For example if I had 12 items first page shows 10, and 2nd pages shows 2. I want all 12 items to be in shown in header(total) row.

**Comment by TeodorKolev** [comment]
Can you give me any example of for cycle adding index before pagination and counting all items in group. Thanks a lot!

**Comment by TeodorKolev** [comment]
In other words counting in group counts the rows in group.

**Comment by TeodorKolev** [comment]
Is there any option to display group total and pagination together?
